### PR TITLE
Remove global mentions

### PR DESCRIPTION
**Pull request by ngrayluna**
Removes "most" reference to global functions. Examples that were not modified need a deeper dive so as to not break the code example.

Note: Code snippets, particularly in `integrations/*`, need revisiting in general. It's unclear how useful they are since they omit key pieces to make the code useable and/or informative.

### CODE REVIEW

**Review by dmitryduev — APPROVED** [review]
much like

File: content/en/guides/integrations/pytorch-geometric.md [hunk 1]
```diff
@@ -81,25 +81,24 @@ The following snippet shows how you could do that with PyVis and HTML.
 
 ```python
 from pyvis.network import Network
-Import wandb
+import wandb
 
-wandb.init(project=’graph_vis’)
-net = Network(height="750px", width="100%", bgcolor="#222222", font_color="white")
+with wandb.init(project=’graph_vis’) as run:
+    net = Network(height="750px", width="100%", bgcolor="#222222", font_color="white")
```
Comment (mdlinville) [inline content/en/guides/integrations/pytorch-geometric.md:87]: I wonder if we should consider standardizing on two spaces per tab in the future to reduce the width. This and a few others use 4.

File: content/en/guides/models/tables/visualize-tables.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ To view the two tables side-by-side, change the first dropdown from "Merge Table
 
 View how values you log to a table change throughout your runs with a step slider. Slide the step slider to view the values logged at different steps. For example, you can view how the loss, accuracy, or other metrics change after each run. 
 
-he slider uses a key to determine the step value. The default key for the slider is `_step`, a special key that W&B automatically logs for you. The `_step` key is an integer that increments by 1 each time you call `wandb.log()` in your code.
+he slider uses a key to determine the step value. The default key for the slider is `_step`, a special key that W&B automatically logs for you. The `_step` key is an integer that increments by 1 each time you call `wandb.Run.log()` in your code.
```
Comment (mdlinville) [inline content/en/guides/models/tables/visualize-tables.md]:
```suggestion
The slider uses a key to determine the step value. The default key for the slider is `_step`, a special key that W&B automatically logs for you. The `_step` key is an integer that increments by 1 each time you call `wandb.Run.log()` in your code.
```

File: content/en/guides/models/track/log/log-summary.md [hunk 3]
```diff
@@ -8,18 +8,21 @@ title: Log summary metrics
 
 In addition to values that change over time during training, it is often important to track a single value that summarizes a model or a preprocessing step. Log this information in a W&B Run's `summary` dictionary. A Run's summary dictionary can handle numpy arrays, PyTorch tensors or TensorFlow tensors. When a value is one of these types we persist the entire tensor in a binary file and store high level metrics in the summary object, such as min, mean, variance, percentiles, and more.
 
- The last value logged with `wandb.log` is automatically set as the summary dictionary in a W&B Run. If a summary metric dictionary is modified, the previous value is lost.
+ The last value logged with `wandb.Run.log()` is automatically set as the summary dictionary in a W&B Run. If a summary metric dictionary is modified, the previous value is lost.
```
Comment (mdlinville) [inline content/en/guides/models/track/log/log-summary.md]:
```suggestion
The last value logged with `wandb.Run.log()` is automatically set as the summary dictionary in a W&B Run. If a summary metric dictionary is modified, the previous value is lost.
```

File: content/en/support/kb-articles/pass_class_attribute_wandblog.md [hunk 4]
```diff
@@ -1,9 +1,9 @@
 ---
 url: /support/:filename
-title: "What happens if I pass a class attribute into wandb.log()?"
+title: "What happens if I pass a class attribute into run.log()?"
```
Comment (mdlinville) [inline content/en/support/kb-articles/pass_class_attribute_wandblog.md]: Should this be the capital one?

File: content/en/support/kb-articles/plot_multiple_lines_plot_legend.md [hunk 5]
```diff
@@ -10,13 +10,16 @@ support:
...
 
 ```python
-wandb.log(
-    {
-        "my_plot": wandb.plot.line_series(
-            xs=x_data, ys=y_data, keys=["metric_A", "metric_B"]
-        )
-    }
-)
+
```
Comment (mdlinville) [inline content/en/support/kb-articles/plot_multiple_lines_plot_legend.md:13]:
```suggestion
```

File: content/en/tutorials/artifacts.md [hunk 6]
```diff
@@ -443,7 +443,7 @@ class ConvNet(nn.Module):
 ```
 
 Here, we're using W&B to track the run,
-and so using the [`wandb.config`](https://colab.research.google.com/github/wandb/examples/blob/master/colabs/wandb-config/Configs_in_W%26B.ipynb)
+and so using the [`run.config`](https://colab.research.google.com/github/wandb/examples/blob/master/colabs/wandb-config/Configs_in_W%26B.ipynb)
```
Comment (mdlinville) [inline content/en/tutorials/artifacts.md:446]: It's not letting me make a suggestion here but you could lose the comma and the "so".

File: content/en/tutorials/experiments.md [hunk 7]
```diff
@@ -47,41 +47,32 @@ Create, track, and visualize a machine learning experiment. To do this:
...
 
       # 2️. Log metrics from your script to W&B
-      wandb.log({"acc": acc, "loss": loss})
-
-  # Mark the run as finished
-  wandb.finish()
+      run.log({"acc": acc, "loss": loss})    
 ```
 
+
```
Comment (mdlinville) [inline content/en/tutorials/experiments.md:75]:
```suggestion
```

**Review by mdlinville — APPROVED** [review]
Just a few nits for followup
